packages/gatsby/src/utils/generic-on-error.ts

```diff
@@ -0,0 +1,6 @@
+import reporter from "gatsby-cli/lib/reporter"
+
+export function genericOnError(_ctx, event): void {
```

Comment:
You can use reporter from context
```suggestion
export function genericOnError({ reporter }, event): void {
```